universe/node/api_skybox.go

```diff
@@ -542,7 +542,7 @@ func (n *Node) apiRemoveSkyboxByID(c *gin.Context) {
 	c.JSON(http.StatusOK, id)
 }
 
-func (n *Node) getApiKeyAndSecret() (*string, *string, error) {
+func (n *Node) getApiBlockadelabsKeyAndSecret() (*string, *string, error) {
```

Comment:
haha, good example of the huuuuge node package and potential conflicts :)
Not a blocker, just a reminder for the future to move stuff outside node package.

Reply:
It is good to refactor the whole Node pkg in one go... but i think it will be quite some work keeping conflicts in mind